include/experimental/__p0009_bits/config.hpp

```diff
@@ -124,6 +126,15 @@ static_assert(_MDSPAN_CPLUSPLUS >= MDSPAN_CXX_STD_14, "mdspan requires C++14 or
 #  endif
 #endif
 
+// NVCC older than 11.6 chokes on the no-unique-address-emulation
+// so just pretend to use it, and leave the macro empty
```

Comment:
Why do we pretend we are using it?

Reply:
because otherwise the code try to uses a complicated EBO workaround which fails horribly with NVCC

Reply:
OK please comment to that effect

Reply:
done